include/dba_pdo.php

```diff
@@ -1,20 +1,24 @@
 <?php
 
+use DDDBL;
+
+use Exception;
+
 require_once('include/datetime.php');
 
-$objDDDBLResultHandler = new \DDDBL\DataObjectPool('Result-Handler');
+$objDDDBLResultHandler = new DDDBL\DataObjectPool('Result-Handler');
```

Comment:
You were right:
`PHP Warning:  The use statement with non-compound name 'DDDBL' has no effect in include/dba_pdo.php on line 3`